common/persistence/src/main/resources/db/migration/V7__createFederationUploadKeyTable.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE federation_upload_key (
+    key_data bytea PRIMARY KEY,
+    rolling_period integer NOT NULL,
+    rolling_start_interval_number integer NOT NULL,
+    submission_timestamp bigint NOT NULL,
+    transmission_risk_level integer NOT NULL,
+    consent_to_federation boolean NOT NULL DEFAULT FALSE,
+    origin_country varchar (2),
+    visited_countries varchar (2) [],
+    report_type varchar(30),
```

Comment:
Same as above: Are we expecting a value set of size greater than 1 at any point in time?

Reply:
Are you referring to the visited Countries set? The answer would be yes

Reply:
I am referring to `report_type`. I assume the answer is the same as for `origin_country`.

Reply:
That's correct